source/includes/tutorial1/_inkling.html.md

```diff
@@ -0,0 +1,135 @@
+
+A curriculum contains information the Bonsai AI Engine uses to train your BRAIN on the concept you've specified. They also specify the reward function (`objective`) for teaching a given concept. The reward function is a way the system concretely measures the AI's performance as it learns the concept. For this tutorial we will only using one concept, and therefore, one curriculum. 
+
+Every lesson must have a configuration. In this case, we have a simple example that doesn’t need a configuration, so the config schema is named `dummy` with a default value of `-1` which in this case does nothing to configure the lesson.
+
+For more information about using these keywords, refer to the [Curriculums Reference][4] and [Lessons Reference][5].
+
+# The Inkling/Simulation Relationship
+
+Your Inkling code and your simulation are tightly coupled -- through the SDK bridge class. The Inkling describes what to expect from the simulation as state, and what actions and configurations to send to the simulation. The bridge class does any conversion needed to make this match the simulation. This section describes how the different parts of your Inkling program and your simulation relate to each other through the bridge. The colors in the table below indicate which parts are connected. 
+
+> ![Inkling/Simulator Graphic](../images/tutorial1-inksim.png)
+
+| Color               | Description  |
+| -                   | -  |
+| Purple (dark/light) | The Inkling state schema field names and types must match the state dictionaries returned from `episode_start` and `simulate` in the simulator. |
+| Blue (dark/light)   | The Inkling action schema field names will match the keys in the action dictionary passed to `simulate` in the simulator, and the values will have the types specified in Inkling, and will obey the specified constraints (`{-1, 0, 1}` in the example). |
+| Orange (dark/light) | The simulator's configuration passes as `parameters` to the `episode_start`, and will take values from the `constrain` clause in Inkling (if used). |
+| Red                 | The name of the concept must match the `train` clause in the curriculum for that concept. |
+| Green               | The simulator name must match between the `simulator` clause and the `with simulator` clause in the curriculum. The simulator must pass the same name to the constructor of the `Simulator` class, so the AI engine knows which simulator is connected. |
+|Turquoise            | The name of the optimization objective or reward function appears twice in the Inkling, and is available as `self.objective_name` in the simulator. |
+
+<aside class="notice">
+Note that config in __main__ is the brain configuration and remains the same throughout, whereas goal_config (highlighted in orange) is used at the beginning of every episode and must be named the same as it is in Inkling. These configs are unrelated.
```

Comment:
do `config` and `__main__` and `goal_config` need backticks?

Reply:
I am unable to do code formatting in the HTML notices unfortunately.